src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -654,7 +657,13 @@ WhisperDecodedResults WhisperPipeline::StaticWhisperPipeline::generate(
 
         // prepare init_ids just once for whole input
         if (init_ids.empty()) {
+            OPENVINO_ASSERT(m_models.decoder.get_tensor("input_ids").get_shape().back() == 1);
```

Comment:
What does  this check do?

Reply:
I assume it can't be incorrect, as we explicitly reshape model for `input_ids` size.

Besides, I'd rather check: `get_tensor("input_ids").get_size()`